WTWSMS/localisation/WtWSMS_religions.csv

```diff
@@ -22,24 +22,25 @@ d_binitarian;Binitarian Patriarchy;;;;;;;;;;;x
 pelagianism;Pelagian;;;;;;;;;;;x
 pelagianism_DESC;Pelagianism is the belief that original sin did not taint human nature and that mortal will is still capable of choosing good or evil without special Divine aid. This is still sometimes called Limited Depravity. Thus, in contrast to 1 Corinthians 15:19-22, Adam's sin was "to set a bad example" for his progeny, but his actions did not have the other consequences imputed to original sin. Pelagianism views the role of Jesus as "setting a good example" for the rest of humanity as well as providing an atonement for our sins. In short, humanity has full control, and thus full responsibility, for obeying the Gospel in addition to full responsibility for every sin. According to Pelagian doctrine, because humans are sinners by choice, they are therefore criminals who need the atonement of Jesus Christ. Sinners are not victims, they are criminals who need pardon.;;;;;;;;;;;x
 antinomianism;Antinomian;;;;;;;;;;;x
-antinomianism_DESC;Any view which holds that Christians are freed by grace from obligations of any moral law. St Paul had to refute a charge of this type made by opponents because of his attitude to the Mosaic Law ;;;;;;;;;;;x
+antinomianism_DESC;Any view which holds that Christians are freed by grace from obligations of any moral law, and thus can sin at will if they ask for forgiveness. Saint Paul had to refute a charge of this type made by opponents because of his attitude to the Mosaic Law.;;;;;;;;;;;x
+antinomianism_FEATURES;\n�Y*�! Antinomians can commit incest, as long as they ask for forgiveness afterwards \n�Y*�! Antinomians do not reject the pleasures of the flesh, as long as they ask for forgiveness afterwards \n�Y*�! Antinomians are shunned by other Christians, and thus tend to rely on each other more
 ebionites;Ebionite;;;;;;;;;;;x
 ebionites_DESC;Ebionites, or Ebionaioi, is a patristic term referring to a Jewish Christian movement that existed during the early centuries of the Christian Era. They regarded Jesus of Nazareth as the Messiah while rejecting his divinity and insisted on the necessity of following Jewish law and rites. The Ebionites used only one of the Jewish Gospels, revered James the Just and rejected Paul of Tarsus as an apostate from the Law. Their name suggests that they placed a special value on voluntary poverty. They have been identified as the meek and the poor Jesus spoke of in the Beatitudes, and the Ebionim was one of the terms used by the sect that sought to separate themselves from the corruption of the Temple, at Qumran, whom many believe were the Essenes.;;;;;;;;;;;x
 d_ebionites;The Ebionite Church;;;;;;;;;;;x
-d_ebionites_adj;Jewish;;;;;;;;;;;x
-d_ebionites_ADJ;Rabbinical;;;;;;;;;;;x
+d_ebionites_adj;Ebionaioi;;;;;;;;;;;x
+d_ebionites_ADJ;Ebionaioi;;;;;;;;;;;x
```

Comment:
@loup99 

-because there's a difference between Ebionites and Orthodox Jews, even though they are both extremely strict about Jewish law. Also, if someone were to restore the High Priesthood with Judaism and then get knocked off by the Ebinotes, who then restored their High Priesthood, things would get weird if both said "JEWISH".

Even so, I was hoping, one day, to code a Decision that could allow Ebionites to be moved to the Jewish group (similar to how Catholics can be forced to move into the Orthodox family) and set Judaism as a heresy of the Ebionites.